test/E2ETests/Common/DeploymentUtility.cs

```diff
@@ -201,7 +201,7 @@ private static Process StartMonoHost(StartParameters startParameters, ILogger lo
 
             var hostProcess = Process.Start(startInfo);
             logger.WriteInformation("Started {0}. Process Id : {1}", hostProcess.MainModule.FileName, hostProcess.Id);
-            Thread.Sleep(15 * 1000);
+            Thread.Sleep(25 * 1000);
```

Comment:
We are basically waiting for Kestrel to start. I can open a bug on Kestrel to provide some notification on start